src/llmcompressor/transformers/finetune/text_generation.py

```diff
@@ -83,19 +83,17 @@ def eval(**kwargs):
     main(model_args, data_args, recipe_args, training_args)
 
 
-def oneshot(**kwargs):
-    """
-    CLI entrypoint for running oneshot calibration
-    """
-    # TODO: Get rid of training args when Oneshot refactor comes in
-    model_args, data_args, recipe_args, training_args = parse_args(**kwargs)
-    training_args.do_oneshot = True
-
-    main(model_args, data_args, recipe_args, training_args)
-
+def oneshot(**kwargs) -> None:
+    logger.warning(
```

Comment:
Let's use this: 
https://github.com/neuralmagic/compressed-tensors/blob/5f24384752a6b5e6c94ce9ba949213a403b46947/src/compressed_tensors/utils/helpers.py#L169

Reply:
```python3
message = (
                f"{func.__name__} is deprecated and will be removed in a future release"
            )
            if future_name is not None:
                message += f". Please use {future_name} instead."
```

if I use that i will have "oneshot is deprecated and will be removed in a future release. Please use oneshot instead". 
Should modify the ct code or just keep what I have here.

Reply:
I also don't think the message as is makes sense. As is (and I think we should change this), the llmcompressor.transformers.oneshot import path was removed. (Right now) the only way a user can access this is through llmcompressor.transformers.finetune.text_generation.